packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -126,7 +127,9 @@ export default function Sidebar({
   setOpenMobileSidebar,
   onShowDndClick,
 }: SidebarProps): ReactElement {
-  const activePage = activePageProp === '/' ? '/popular' : activePageProp;
+  const [defaultFeed] = usePersistentState('defaultFeed', null, 'popular');
```

Comment:
Now I see that it is used here as well. Not a good practice all these values will not be in sync during the session and it could create some real mess.